packages/hint-leading-dot-classlist/tests/fixtures/testPassing.js

```diff
@@ -0,0 +1 @@
+document.getElementById('container').classList.add('foo')
```

Comment:
Let's add a test to verify we pass for non-classList objects calling `add` with a leading `.`.
```ts
const set = new Set();
set.add('.foo');
```